config/puma.cfg

```diff
@@ -11,7 +11,6 @@ def get_workers()
 end
 
 preload_app!
-nakayoshi_fork
```

Comment:
@skateman was this removed intentionaly?

Reply:
See [here](https://github.com/puma/puma/blob/master/6.0-Upgrade.md#upgrade) point 3 in the bottom.